### ISSUE TITLE
no info about int division (unlike rstan)

### ISSUE THREAD

**Issue by bnicenboim** [stan-dev/cmdstanr#457]
**Describe the bug**

rstan informs about a potential problem with int division, but cmdstanr doesn't do it even in the pedantic mode.

**To Reproduce**
```
stan_file <- write_stan_file("
data {
}
transformed data {
}
parameters {
  real mu;
  real sigma;
}
model {
  target += normal_lpdf(mu | 0, 1) + 1/2;
}
")

m <- cmdstan_model(stan_file, pedantic = TRUE)
fit <- m$sample(
                 data = list(),
                 seed = 123,
                 parallel_chains = 4,

  )
```
**Expected behavior**

I would expect a message
```
Info: Found int division at 'string', line 9, column 37 to column 38:
  1 / 2
Values will be rounded towards zero. If rounding is not desired you can write
the division as
  1.0 / 2
```

**Operating system**
Ubuntu 20.04.2 LTS

**CmdStanR version number**
0.3.0.9000 (and latest cmdstan)

**Comment by rok-cesnovar** [comment]
Thanks @bnicenboim!

This is related to https://github.com/stan-dev/stanc3/issues/824
Essentially this warning is printed differently than all the others (this one to stdout not stderr as it should be). We will fix this properly for 2.27 hopefully but we can also apply a fix in cmdstanr (by printing stdout during Stan to C++ translation).

**Comment by rok-cesnovar** [comment]
This was fixed on stanc3 master and will appear in the next cmdstan release and there is no intervention needed in cmdstanr after the release so I am going to close this one. 

Thanks again Bruno!
